### DISCUSSION TITLE
Serverless Devs 吐槽小会

### DISCUSSION THREAD

**Discussion by anycodes** [Serverless-Devs/Serverless-Devs#321] [General]
各位小伙伴，Serverless Devs 已经发布了快一年了，大家对 Serverless Devs 有什么期待或者吐槽么？欢迎大家在这个 Discussion 下面留言讨论。

我会随机在留言的用户中，抽取 50% 的用户送小礼物。大家快来一起吐槽啊。

【你的吐槽，就是Serverless Devs进步的基石】

**Comment by anycodes** [comment]
- 缺少 IDE 插件，IDE 插件在使用的时候还是非常重要的
- Desktop 没有详细的帮助文档
- 有一些细节并不规整，例如中英混合，给人很不专业的感觉

**Comment by Aug-Spark** [comment]
官网各种帮助文档配图再多点就好了，有时候没图一时半会摸不着南北，不知道对应的是哪里，俗话说有图有真相。
实践教程也再多点就好了。

**Comment by GotzeWong** [comment]
与云效流水线兼容性希望能尽快加强些，会遇到云效提示“成功”却没有更新到函数计算的情况。

**Reply by heimanba** [reply]
在新版本 2.0.93已经发，麻烦您看下新版本是否还有此现象哈

**Comment by Milky-Liu** [comment]
希望Serverless Devs帮助文档能够继续完善，对小白再友好一点，嘻嘻，能够有更多福利就更好了哈哈

**Comment by llkevin13579** [comment]
文档的搜索功能建议用Algolia，而不是像目前这样嵌入百度搜索，有点风格不搭，Algolia用户挺多的，搜索的效率也挺不错的，建议使用它进行文档的搜索工作，帮助开发者更快地找到所需内容
